manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -186,6 +196,8 @@ spec:
           - name: ISTIO_META_CLUSTER_ID
             value: "{{ $.Values.global.multiCluster.clusterName | default `Kubernetes` }}"
           volumeMounts:
+          - name: istioproxyconfig
```

Comment:
nit: make this `istio-envoy` to align with injection template. If there is a compelling reason not to it should maybe be istio-proxy-config to at least align with the formatting?

Reply:
Done!